### PR TITLE
Created troubleshooting guide and categories

### PR DESCRIPTION
**Pull request by AmieDD**
Created troubleshooting guide.

### CODE REVIEW

**Comment by AmieDD** [comment]
@khaines I'll need your input for creating troubleshooting guides for Metrics. I went ahead and setup a folder so we have it has a placeholder for now.

File: docs/troubleshooting/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# thundernetes Troubleshooting Guide
+
+This is a public repository for all of thundernetes Troublesooting guides, and is intended to provide a central location for community driven troubleshooting content.
```
Comment (dgkanatsios) [inline docs/troubleshooting/README.md:3]: nit: Troubleshooting

File: docs/troubleshooting/Cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This contains the troublshooting Clusters related to thundernetes
```
Comment (dgkanatsios) [inline docs/troubleshooting/Cluster/README.md:1]: nit: troubleshooting

File: docs/troubleshooting/Deployment/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This contains the troublshooting Deployment related to thundernetes
```
Comment (dgkanatsios) [inline docs/troubleshooting/Deployment/README.md:1]: nit: troubleshooting

File: docs/troubleshooting/GameServers/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+This contains the troublshooting Game Servers related to thundernetes
```
Comment (dgkanatsios) [inline docs/troubleshooting/GameServers/README.md:1]: troubleshooting

File: docs/troubleshooting/Kubernetes/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+This contains the troublshooting Kubernetes related to thundernetes
```
Comment (dgkanatsios) [inline docs/troubleshooting/Kubernetes/README.md:1]: nit: troubleshooting

File: docs/troubleshooting/README.md (hunk 1 above)
Comment (AmieDD) [reply docs/troubleshooting/README.md]: Spell check! Opps! Thanks for the catch!

**Comment by khaines** [comment]
> @khaines I'll need your input for creating troubleshooting guides for Metrics. I went ahead and setup a folder so we have it has a placeholder for now.

for sure! we can iterate on the guides, but thank you for getting the stub content in place

**Review by AmieDD — COMMENTED** [review]
Created troubleshooting guide and resolved spelling errors(I'm only human..mostly)!

**Review by AmieDD — COMMENTED** [review]
Updated spelling errors.

**Review by dgkanatsios — APPROVED** [review]

**Pull request merged**